plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/util/Validator.js

```diff
@@ -33,8 +34,42 @@ Ext.define('NX.util.Validator', {
   },
 
   /**
+   * @private
+   */
+  nxNameRegex : /^[^(_)#A-Z\\\/"'\*\?<>|\s,.][^#A-Z\\\/"'\*\?<>|\s,.]*$/,
```

Comment:
I'd rather just disallow them so we don't need to worry about it, but wanted to get some feedback before discounting that we might want to allow some and percent-encode others.
Anyone else feel strongly on this?